pkgs/by-name/us/usbkvm/package.nix

```diff
@@ -94,7 +103,7 @@ stdenv.mkDerivation {
     homepage = "https://github.com/carrotIndustries/usbkvm";
     description = "Open-source USB KVM (Keyboard, Video and Mouse) adapter";
     changelog = "https://github.com/carrotIndustries/usbkvm/releases/tag/v${version}";
-    license = lib.licenses.gpl3;
+    license = lib.licenses.gpl3Plus;
```

Comment:
I received clarification from @carrotIndustries that they want `GPL-v3.0` (not `GPL-v3.0+`) to apply here, and they will update the repository accordingly. Can you change this back, please? Sorry about the back-and-forth.